hide/comp/SceneEditor.hx

```diff
@@ -758,21 +760,24 @@ class SceneEditor {
 							objects3d[i].applyPos(sceneObjs[i]);
 						}
 						refreshProps();
-					}
-					else {
+					} else {
 						for(i in 0...objects3d.length) {
 							objects3d[i].loadTransform(newState[i]);
 							objects3d[i].applyPos(sceneObjs[i]);
 						}
 						refreshProps();
 					}
-
 					for(o in objects3d)
 						o.updateInstance(getContext(o));
 				}));
-
 				for(o in objects3d)
 					o.updateInstance(getContext(o));
+        for(i in 0...objects3d.length) {
```

Comment:
Why not putting this in the `for(o in objects3d)` above ?

Reply:
Let me fix this for you,